samples/cellular/nrf_provisioning/README.rst

```diff
@@ -0,0 +1,141 @@
+.. _nrf_provisioning_sample:
+
+Cellular: nRF Device provisioning
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the :ref:`lib_nrf_provisioning` service on your device.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9161dk_nrf9161_ns
+
+.. include:: /includes/tfm.txt
+
+The sample requires that the device's private key is installed on the device and the associated device UUID is obtained from the Identity Service.
+
+.. note::
+   This sample requires modem firmware v2.0.0 or later.
+
+Overview
+********
+
+The sample shows how the device performs the following actions:
+
+* Connects to nRF Cloud Provisioning Service.
+* Fetches available device-specific provisioning configuration.
+* Decodes the commands.
+* Acts on any AT commands, if available.
+* Reports the results back to the server.
+  In the case of an error, stops processing the commands at the first error and reports it back to server.
+* Sends ``FINISHED`` response if all the previous commands are executed without errors provided and ``FINISHED`` is one of the set provisioning commands.
+
+User interface
+**************
+
+Device side interaction is not required.
+You must define the provisioning configuration at the server side.
+See `nRF Cloud provisioning configuration`_.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following configuration options for the sample:
+
+.. _CONFIG_NRF_PROVISIONING_ROOT_CA_SEC_TAG:
+
+CONFIG_NRF_PROVISIONING_ROOT_CA_SEC_TAG
+   Root CA security tag for the nRF Cloud Provisioning Service.
+   Needs to be set explicitly and if not, the compilation fails.
+
+.. _CONFIG_NRF_PROVISIONING_HTTP_HOSTNAME:
+
+CONFIG_NRF_PROVISIONING_HTTP_HOSTNAME
+   Configures the hostname of the nRF Device provisioning service.
+
+.. _CONFIG_NRF_PROVISIONING_HTTP_PORT:
+
+CONFIG_NRF_PROVISIONING_HTTP_PORT
+   Configures the HTTP port of the nRF Device provisioning service.
+
+.. _CONFIG_NRF_PROVISIONING_HTTP_TIMEOUT_MS:
+
+CONFIG_NRF_PROVISIONING_HTTP_TIMEOUT_MS
+   Configures the HTTP timeout.
+
+.. _CONFIG_RF_PROVISIONING_HTTP_RX_BUF_S:
+
+CONFIG_RF_PROVISIONING_HTTP_RX_BUF_S
+   Configures the response payload buffer size.
+
+.. _CONFIG_NRF_PROVISIONING_HTTP_TX_BUF_SZ:
+
+CONFIG_NRF_PROVISIONING_HTTP_TX_BUF_SZ
+   Configures the command request body size.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/cellular/nrf_provisioning`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Testing
+=======
+
+|test_sample|
+
+#. |connect_kit|
+#. |connect_terminal|
+#. Add a provisioning configuration using the nRF Cloud Provisioning Service.
+#. See `nRF Cloud provisioning configuration`.
```

Comment:
Sorry for a further nitpick, but I think this will now render as a numbered step when it was probably intended to be a follow-up to the previous line. 

@peknis, I don't know the proper formatting to suggest. What do you think?

Reply:
Otherwise, LGTM.